RileyLinkKit/Extensions/NSDate.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  NSDate.swift
+//  RileyLink
+//
+//  Created by Pete Schwamb on 5/18/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+
+public func ==(lhs: NSDate, rhs: NSDate) -> Bool {
```

Comment:
I brought this in to handle some date comparisons that you had written, but I couldn't find out where you had implemented this.  

When I try to make the funcs or the extension itself private, the compiler doesn't accept it.